golang/go-cloud-run-hello-world/Dockerfile

```diff
@@ -16,18 +13,18 @@ COPY . ./
 # Compile the application to /app.
 RUN go build -mod=readonly -v -o /app
 
+# Now create separate deployment image
+FROM gcr.io/distroless/base
+
+# Definition of this variable is used by 'skaffold debug' to identify a golang binary.
+# Default behavior - a failure prints a stack trace for the current goroutine.
+# See https://golang.org/pkg/runtime/
+ENV GOTRACEBACK=single
+
 # Copy template & assets
-COPY index.html /index.html
-COPY assets ./assets/
-
-# If you want to use the debugger, you need to modify the entrypoint to the
-# container and point it to the "dlv debug" command:
-#   * UNCOMMENT the following ENTRYPOINT statement,
-#   * COMMENT OUT the last ENTRYPOINT statement
-# Start the "dlv debug" server on port 3000 of the container.
-ENTRYPOINT ["dlv", "exec", "/app", "--continue", "--accept-multiclient", "--api-version=2", "--headless", "--listen=:3000", "--log"]
-
-# If you want to run WITHOUT the debugging server:
-#   * COMMENT OUT the previous ENTRYPOINT statements,
-#   * UNCOMMENT the following ENTRYPOINT statement.
-# ENTRYPOINT ["/app"]
+WORKDIR /app
+COPY --from=build /app /app/server
+COPY index.html /app/index.html
+COPY assets /app/assets/
+
+ENTRYPOINT ["/app/server"]
```

Comment:
Why do we change the entrypoint from /app to /app/server?

Reply:
It felt wrong to be copying `index.html` and `assets` into the root. It also seemed weird to have `/app/app`.

Reply:
I'm more than happy to change `/app/server` to something else.  `/deploy/app`?

Reply:
I changed this to `/hello-world` and kept the binary as `app`.

Reply:
LGTM - @simonz130 any additional feedback?